src/apps/vpn/mozillavpn.cpp

```diff
@@ -1768,3 +1766,15 @@ void MozillaVPN::registerUrlOpenerLabels() {
                                        : AppConstants::API_STAGING_URL);
   });
 }
+
+// static
+QByteArray MozillaVPN::authorizationHeader() {
```

Comment:
Yay. This is easy to add a test for now...can you add it?

Reply:
yes :)